src/navigation/HomeStack.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from "react"
+import {
+  createStackNavigator,
+  TransitionPresets,
+} from "@react-navigation/stack"
+
+import { HomeScreens, HomeScreen } from "./index"
+import Home from "../Home/Home"
+import BluetoothInfo from "../Home/BluetoothInfo"
+import ProximityTracingInfo from "../Home/ProximityTracingInfo"
+
+type HomeStackParams = {
+  [key in HomeScreen]: undefined
```

Comment:
why is this undefined?

Reply:
I think because none of these screens take params. But actually this can just be removed altogether, so I'll do that.